build_tools/azure/install.sh

```diff
@@ -79,4 +79,4 @@ except ImportError:
     print('pandas not installed')
 "
 pip list
-python setup.py develop
+pip install -e .
```

Comment:
Just remaining from some debugging. I'll remove from this PR, however, I thought that it is the recommended way to install something instead of using `setup.py` but I don't remember the reason

Reply:
Using `python setup.py develop` output to stdout which is useful when looking at what Cython is doing.

Reply:
You can `pip install -v -e .` to get the verbose output.

In the long term, the community should move away from programmatic `setup.py` and go towards declarative `pyproject.toml` (https://www.python.org/dev/peps/pep-0518/) with pluggable build systems. So it's "good" to take the abit to not rely on the existance of a `setup.py` file. However there is a no PEP-578 compatible build system that can build project like scikit-learn that have numpy related cython extensions as far as I know.